nsxt/resource_nsxt_policy_ipsec_vpn_session_test.go

```diff
@@ -0,0 +1,404 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyIPSecVpnSessionRouteBasedCreateAttributes = map[string]string{
+	"display_name":               getAccTestResourceName(),
+	"description":                "Terraform-provisioned IPsec Route-Based VPN",
+	"enabled":                    "true",
+	"vpn_type":                   "RouteBasedIPSecVpnSession",
+	"authentication_mode":        "PSK",
+	"compliance_suite":           "NONE",
+	"subnets":                    "169.254.152.25",
+	"prefix_length":              "24",
+	"peer_address":               "18.18.18.22",
+	"peer_id":                    "18.18.18.22",
+	"psk":                        "VMware123!",
+	"connection_initiation_mode": "INITIATOR",
+}
+
+var accTestPolicyIPSecVpnSessionRouteBasedUpdateAttributes = map[string]string{
+	"display_name":               getAccTestResourceName(),
+	"description":                "Terraform-provisioned IPsec Route-Based VPN",
```

Comment:
as above, it would be preferrable to add some more variation to update, so that the test has more coverage